package.json

```diff
@@ -5,8 +5,8 @@
     "4.0.1"
   ],
   "dependencies": {
-    "de.basisprofil.r4": " 1.4.0",
+    "de.basisprofil.r4": "1.4.0",
```

Comment:
Lass uns in diesem Zuge auf "de.basisprofil.r4": "1.5.x",  gehen. Das ist aktuell der Standard den wir gematikseitig durchsetzen wollen und der auch schon in der ePA und dem E-Rezept übernommen wurde